### ISSUE TITLE
why is get mandatory ?

### ISSUE THREAD

**Issue by bruceauyeung** [foxbenjaminfox/vue-async-computed-decorator#9]
my code:
~~~
  @AsyncComputed({
    default: false,
  })
  async isLoggedIn() {
    return uacAuth.isLoggedIn;
  }
~~~
IAsyncComputedValue definition
~~~
export interface IAsyncComputedValue<T> {
  default?: T | (() => T);
  get: AsyncComputedGetter<T>;
  watch?: string[] | (() => void);
  shouldUpdate?: () => boolean;
  lazy?: boolean;
}
~~~
![image](https://user-images.githubusercontent.com/1210201/75946466-2ed6d700-5ed8-11ea-8a53-4f0291bf7347.png)
in my case , the async method `isLoggedIn` is actually the `getter`, i don't think `get` in IAsyncComputedValue here should be defined as mandatory.

**Comment by bruceauyeung** [comment]
now i know what caused the problem.   command  `yarn add vue-async-computed-decorator` resolves `vue-async-computed-decorator` to version `0.0.4` ,which is not the latest typescript-based version.
i am trying to install the lastest version via `yarn add https://github.com/foxbenjaminfox/vue-async-computed-decorator.git` but unfortunately failed because of my company proxy

**Comment by bruceauyeung** [comment]
`yarn add https://github.com/foxbenjaminfox/vue-async-computed-decorator.git` also resolves `vue-async-computed-decorator` to version `0.0.4`
>\>yarn.cmd add https://github.com/foxbenjaminfox/vue-async-computed-decorator.git
>yarn add v1.22.0
>[1/4] Resolving packages...
>[2/4] Fetching packages...
>info fsevents@1.2.9: The platform "win32" is incompatible with this module.
>info "fsevents@1.2.9" is an optional dependency and failed compatibility check. Excluding it from installation.
>info fsevents@2.1.2: The platform "win32" is incompatible with this module.
>info "fsevents@2.1.2" is an optional dependency and failed compatibility check. Excluding it from installation.
>[3/4] Linking dependencies...
>warning " > jest-electron@0.1.11" has unmet peer dependency "jest@^24.0.0".
>warning " > less-loader@5.0.0" has unmet peer dependency "webpack@^2.0.0 || ^3.0.0 || ^4.0.0".
>[4/4] Building fresh packages...
>success Saved lockfile.
>success Saved 1 new dependency.
>info Direct dependencies
>└─ vue-async-computed-decorator@0.0.4
>info All dependencies
>└─ vue-async-computed-decorator@0.0.4
>

**Comment by DGollings** [comment]
ran into same issue, although running "npm install -S https://github.com/foxbenjaminfox/vue-async-computed-decorator.git" fixed it for me.

Might I suggest releasing a 0.0.5 ?

**Comment by foxbenjaminfox** [comment]
Sorry, it took me some time to get to this, but I've just now released `v0.0.5`.
